bundle/02_mednist_classification.ipynb

```diff
@@ -0,0 +1,608 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import os
</pre><p>I would suggest separating this import part into a <code>Setup imports</code> section and also adding a <code>Setup environment</code> section in case users don't have a proper env that can't run the notebook immediately.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1493/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
This cell isn't meant to be run, it's just the code from the monai_101 tutorial condensed to a single cell.

Reply:
What I can do is put it in the markdown cell instead. That should get around CI check problems too.